catalog/from-k8s/resource_test.go

```diff
@@ -1770,3 +1770,297 @@ func testService(name string) *apiv1.Service {
 		},
 	}
 }
+
```

Comment:
The `testService` function above this is a helper function rather than a test. Usually helper functions are at the bottom of the file, so I'd suggest moving the tests around a bit. Additionally, the standard in this file is to have all of the permutations of `loadBalancer`, `nodePort`, etc. tests are grouped together, so it would make more sense to insert these tests near their service type.